test/fixtures/moves.json

```diff
@@ -80,14 +74,8 @@
             "identifier_type": "police_national_computer",
             "value": "11223344"
           },
-          {
-            "identifier_type": "prison_number",
-            "value": "796507"
-          },
-          {
-            "identifier_type": "athena_reference",
-            "value": "763307"
-          }
+          { "identifier_type": "prison_number", "value": "796507" },
```

Comment:
Once again, and this is nothing to do with this PR, I feel that what's being tested is somewhat bogus.

If we're running against v2, the identiers shouldn't be present and these props are actually flat on the person obj.

Also, again hating the (to my mind) unnecessary document reflow caused by prettier line length  pfft pfft pfft...